egs2/librimix/asr1/RESULTS.md

```diff
@@ -0,0 +1,30 @@
+<!-- Generated by scripts/utils/show_asr_result.sh -->
+# Transformer multispkr
+## Environments
+- date: `Thu Nov 10 14:58:09 EST 2022`
+- python version: `3.9.13 (main, Aug 25 2022, 23:26:10)  [GCC 11.2.0]`
+- espnet version: `espnet 202209`
+- pytorch version: `pytorch 1.12.1`
+- Git hash: `b3c185d5d707bb385b74f42df2cc59bcf7d7e754`
+  - Commit date: `Wed Nov 9 22:00:30 2022 -0500`
+- ASR config: [conf/tuning/train_asr_transformer_multispkr.yaml](conf/tuning/train_asr_transformer_multispkr.yaml)
+- Decode config: [conf/decode.yaml](conf/decode.yaml)
+- Pretrained model: [https://huggingface.co/espnet/simpleoier_librimix_asr_train_asr_transformer_multispkr_raw_en_char_sp](https://huggingface.co/espnet/simpleoier_librimix_asr_train_asr_transformer_multispkr_raw_en_char_sp)
+
+## asr_train_asr_transformer_multispkr_raw_en_char_sp
+### WER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_multi_asrtrue_lm_lm_train_lm_transformer_en_char_valid.loss.ave_asr_model_valid.acc.ave/test|6000|111243|80.4|17.4|2.2|3.8|23.5|88.0|
+
+### CER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
+|decode_multi_asrtrue_lm_lm_train_lm_transformer_en_char_valid.loss.ave_asr_model_valid.acc.ave/test|6000|590408|90.5|6.1|3.5|3.9|13.5|88.0|
+
+### TER
+
+|dataset|Snt|Wrd|Corr|Sub|Del|Ins|Err|S.Err|
+|---|---|---|---|---|---|---|---|---|
```

Comment:
You could remove these lines if there is no result.

Reply:
done.